charts/prometheus-json-exporter/values.yaml

```diff
@@ -24,6 +24,9 @@ serviceAccount:
   # If not set and create is true, a name is generated using the fullname template
   name: ""
 
+# Annotations for the Deployment
+annotations: {}
```

Comment:
Considering the scope is the deployment, should this either be named `deploymentAnnotations`, to follow what we have for pods (`podAnnotations`), or rather create:
```
deployment:
  annotations: {}
```
to put all the deployment specific values in the future? Or should this field be a set of common annotations on all resources?

Reply:
I agree, if we name it `annotations` it should apply to all resources. 

I renamed it to `deploymentAnnotations` to have it consistent with `podAnnotations`.